doc/docs/master/how-tos/developer-workflow/index.md

```diff
@@ -0,0 +1,32 @@
+# Developer Workflow
+
+In general, the developer workflow for Pachyderm involves adding 
+data to versioned data repositories and adding a pipelines that 
+read from data repositories to execute your code against that data. 
```

Comment:
```suggestion
transforms that data via code you've written.
```

Reply:
Not quite sure how to reword this, but I think it's valuable to ensure the user doesn't think the data repositories are read-only